lib/main_common.pm

```diff
@@ -2401,116 +2401,186 @@ sub load_virtualization_tests {
-        loadtest "virtualization/universal/stresstest";    # Perform stress tests on the guests
-        loadtest "console/perf";
-        loadtest "console/oprofile" unless (get_var("REGRESSION", '') =~ /xen/);
-    }
+        # QAM VIRT irqbalance test
+        unless (get_var('DISABLE_IRQBALANCE_TEST')) {
+            if (check_var('HOST_HYPERVISOR', 'xen')) {
+                loadtest "console/login";
+                loadtest "virt_autotest/cleanup_libvirtd_log";
+                loadtest "virt_autotest/xen_guest_irqbalance";
+                loadtest "virtualization/universal/finish";
+            }
+        }
 
-    if ($windows) {
-        loadtest "virtualization/universal/download_image";    # Download Windows disk image(s)
-        loadtest "virtualization/universal/windows";    # Import and test Windows
+        # QAM VIRT networking test
+        unless (get_var('DISABLE_NETWORKING_TEST')) {
+            loadtest "console/login";
+            loadtest "virt_autotest/cleanup_libvirtd_log";
+            loadtest "virt_autotest/libvirt_host_bridge_virtual_network";
+            loadtest "virt_autotest/libvirt_nated_virtual_network";
+            loadtest "virt_autotest/libvirt_isolated_virtual_network";
+            loadtest "virtualization/universal/finish";
+        }
+
+        # QAM VIRT virtmanager test
+        unless (get_var('DISABLE_VIRTMANAGER_TEST')) {
+            loadtest "console/login";
+            loadtest "virt_autotest/cleanup_libvirtd_log";
+            loadtest "virtualization/universal/virtmanager_init";
+            loadtest "virtualization/universal/virtmanager_offon";
+            loadtest "virtualization/universal/virtmanager_add_devices";
+            loadtest "virtualization/universal/virtmanager_rm_devices";
+            loadtest "virtualization/universal/finish";
+        }
+
+        # QAM VIRT hotplugging test
+        unless (get_var('DISABLE_HOTPLUGGING_TEST')) {
+            loadtest "console/login";
+            loadtest "virt_autotest/cleanup_libvirtd_log";
+            loadtest "virtualization/universal/hotplugging_guest_preparation";
+            loadtest "virtualization/universal/hotplugging_network_interfaces";
+            loadtest "virtualization/universal/hotplugging_HDD";
+            loadtest "virtualization/universal/hotplugging_vCPUs";
+            loadtest "virtualization/universal/hotplugging_memory";
+            loadtest "virtualization/universal/hotplugging_cleanup";
+            loadtest "virtualization/universal/finish";
+        }
+
+        # QAM VIRT snapshots test
+        unless (get_var('DISABLE_SNAPSHOTS_TEST')) {
+            if (check_var('HOST_HYPERVISOR', 'kvm')) {
+                loadtest "console/login";
+                loadtest "virt_autotest/virsh_internal_snapshot";
+                loadtest "virt_autotest/virsh_external_snapshot";
+                loadtest "virtualization/universal/finish";
+            }
+        }
```

Comment:
We already use following settings styles to turn on/off features testing. Could you help make corresponding changes ?

```
ENABLE_HOTPLUGGING
ENABLE_STORAGE
ENABLE_SNAPSHOT
```

For example, ```ENABLE_IRQBALANCE```.

Reply:
Checking.

Reply:
I checked such settings styles, still need add such setting on job group or test suite, while it don't need make such setting like DISABLE_XXX=0.
DISABLE_XX is for debugging when trigger job command manually.
Default is enabled.

Reply:
updated with the similar setting. like: 
ENABLE_HOTPLUGGING
ENABLE_STORAGE
ENABLE_SNAPSHOT